src/backend/distributed/citus--6.1-5--6.1-6.sql

```diff
@@ -0,0 +1,12 @@
+/* citus--6.1-5--6.1-6.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION get_shard_id_for_distribution_column(table_name regclass, distribution_value anyelement)
```

Comment:
Is anyelement really the right thing here? Why aren't arrays acceptable keys? I suspect this actually should be any, rather than anyelement.